pkg/ovirt/configFuncs.es6

```diff
@@ -106,6 +107,23 @@ function doReadHostname ({ dispatch }) {
             });
 }
 
+function doReadIpAddresses ({ dispatch }) {
+    let output = '';
+    return cockpit.spawn(['hostname', '--all-ip-addresses'], {'err': 'message'})
+            .stream(data => {
+                output += data;
+            })
```

Comment:
FYI, for processes that have only very little output, such as this or the `hostname -f` above, there is no need to have a `.stream` handler and accumulate the data. Just get the entire output in the `.done()` handler. That makes the code easier to read and saves memory and runtime on top.

However, it's not wrong like this, so this can be a cleanup follow-up.